.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```

Comment:
Don't check in the the .idea directory, the build directory, the dist directory, or the generated resources.